File: S_3/Programação Orientada a Objetos/Exercicio4.java

import java.io.IOException;

public class Exercicio4 {

    public static void main(String[] args) throws IOException {
        System.out.println("Digite um inteiro:");
        int x = EntradaTeclado.leInt();
        int m = x;
        while (x > 0) {
            int k = x;
            int j = m - x;
            while (j > 0) {
                System.out.print(" ");
                j--;
            }
            while (k > 0) {

                System.out.print("*");
                k--;
            }
            
            System.out.println("");
            x--;
        }
    }
}

File: S_3/Programação Orientada a Objetos/Exercicio4.py
x = int(input("Digite um inteiro:"))
y = x
j = x
while y > 0:
    x = y
    k = j-x
    while k > 0:
        print(" ",end="")
        k -=1
    while x > 0:
        print("*", end="")
        x-=1
    print("")
    y-=1